automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -101,15 +114,27 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     clusterCheck(cluster, googleProject, clusterName, Seq(ClusterStatus.Creating), clusterRequest)
 
     // verify with get()
-    clusterCheck(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName, Seq(ClusterStatus.Creating), clusterRequest)
+    val creatingCluster = clusterCheck(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName, Seq(ClusterStatus.Creating), clusterRequest)
 
     // wait for "Running" or error (fail fast)
     implicit val patienceConfig: PatienceConfig = clusterPatience
-    val actualCluster = eventually {
-      clusterCheck(Leonardo.cluster.get(googleProject, clusterName), googleProject, clusterName, Seq(ClusterStatus.Running, ClusterStatus.Error), clusterRequest)
+    val actualCluster = Try {
```

Comment:
Wrapped in `Try` so even if this fails we download the init bucket logs

Reply:
so the check failing will fail the test but still return a Failure of... some kind of test-failed-exception???

Reply:
We call `.get` on the `Try` below which will re-throw the exception. I just wanted to make `saveClusterInitLogFile` run regardless.